src/frequenz/channels/util/_file_watcher.py

```diff
@@ -39,19 +40,16 @@ class Event:
     def __init__(
         self,
         paths: list[pathlib.Path | str],
-        event_types: set[EventType] | None = None,
+        event_types: abc.Iterable[EventType] = frozenset(EventType),
     ) -> None:
         """Create a `FileWatcher` instance.
 
         Args:
             paths: Paths to watch for changes.
-            event_types: Types of events to watch for or `None` to watch for
+            event_types: Types of events to watch for. Defaults to watch for
```

Comment:
We don't really need to specify the default now, as the documentation renderer will show `frozenset(EventType)` as the default, but just FYI, I think in this case being more explicit is good as it is not trivial to see what `frozenset(EventType)` does.

Reply:
Yes, there was previously an inline comments to clarify that `set(EventType)` sets all event types so I thought it was worth it to explicitly mention it here. I'll keep it then if you don't have a strong opinion against it